dub.json

```diff
@@ -13,17 +13,17 @@
 			"name": "openssl",
 			"targetType": "library",
 			"dependencies" : {
-				"openssl": "~>1.1.4+1.0.1g"
+				"openssl": "~>3.2"
 			},
 			"versions": ["UseOpenSSL"]
 		},
 		{
 			"name": "openssl-1.1",
 			"targetType": "library",
 			"dependencies" : {
-				"openssl": "~>1.1.4+1.0.1g"
+				"openssl": "~>3.2"
 			},
-			"versions": ["UseOpenSSL", "OpenSSL11"]
+			"versions": ["UseOpenSSL"]
```

Comment:
This is now a dupliocate of the `openssl` config above - unfortunately, the `web` plugin specifies the `openssl-1.1` config, so dropping it and special-casing for SIL master is a bit of a PITA.

Reply:
You should depend on the manual configuration and give it a version to keep semantics.

Reply:
We don't need to keep semantics - we're the only users of this repo, and re-using the `openssl-1.1` config makes stuff considerably simpler for us (otherwise we'd have to deal with the `buildForSIL-…` SIL builder stuff, to specify different subconfigurations depending on SIL version the plugin is built again...).

Reply:
Actually, that's not true; we can just use the new `jwtd` version for all SIL versions.

Reply:
Then should this be removed ?

Reply:
It is removed now, yes.